olive/passes/onnx/perf_tuning.py

```diff
@@ -114,6 +119,13 @@ def threads_num_tuning(model, latency_metric, config, tuning_combo):
                 {"trt_fp16_enable": config.trt_fp16_enable},
             )
         ]
+    elif provider == "CUDAExecutionProvider":
```

Comment:
should we add warning if the EP is not cuda but we specify enable_cuda_graph?

Reply:
When enable_cuda_graph is True and provider is not cuda ep, enable_cuda_graph will be ignored, and show some logs.